src/lib/y2network/config.rb

```diff
@@ -150,6 +151,17 @@ def delete_interface(name)
       interfaces.reject! { |i| i.name == name }
     end
 
+    # Adds or update a connection config
+    #
+    # If it is a virtual connection, it adds interface if it does not exist.
```

Comment:
what happens with "fake" interface? should not be also added here?

Reply:
Good point. I will fix it.

Reply:
still it does not mention here that it can also create not attached physical interface. Basically it always create interface if it does not exist already.